test/scripts/compare_with_azarashi.py

```
@@ -147,7 +147,7 @@ def json_serial(obj):
         return {k: json_serial(v) for k, v in obj.__dict__.items()}
     try:
         return str(obj)
-    except:
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)